test/src/test/java/org/corfudb/recovery/Helpers.java

```diff
@@ -72,7 +72,11 @@ static void assertThatMapIsBuilt(CorfuRuntime rt1, CorfuRuntime rt2, String stre
         // Assert that UnderlyingObjects are at the same version
         // If they are at the same version, a sync on the object will
         // be a no op for the new runtime.
-        assertThat(vo1Prime.getVersionUnsafe()).isEqualTo(vo1.getVersionUnsafe());
+
+        // This utility is used by many checkpoint tests and because new checkpoint
+        // inserts a NO_OP entry, VLO version will differ from original (before checkpoint) to
```

Comment:
uncomment test?

Reply:
There were too many. I'll later write another utility to separate the ones that can use this comparison from the ones that can't.